### DISCUSSION TITLE
Is there anyway to test connection migrate?

### DISCUSSION THREAD

**Discussion by Neo-ZK** [alibaba/xquic#14] [Q&A] [answered]
It seem that there is no option for testing connection migrate in test_client/test_server

**Comment by Kulsk** [comment, marked as answer]
migration is not supported

**Comment by DarkPassion** [comment]
@Kulsk  Whether the current version supports connection migration

**Reply by NeutrinoLiu** [reply]
you might want to check https://github.com/alibaba/xquic/blame/07c505142889ea889583d917df88eb63bec47b6e/src/transport/xqc_engine.c#L1214
